### PR TITLE
Use yarn run -T to run command installed at the root of the monorepo

### PR DESCRIPTION
**Pull request by alexandrebodin**
### What does it do?

This PR uses yarn run -T feature to run bins installed at the root level of the repo

### Why is it needed?

With the move to yarn 3 we did not notice that you could do `yarn lint` inside a package but had to use only the root level nx commands. Even if overall it is better to use nx to use caching etc it is still important that the commands work "localy" in the packages. 

### How to test it?

running yarn test:unit/lint/test:front should work inside a package (and not use nx)

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: packages/plugins/color-picker/package.json [hunk 1]
```diff
@@ -31,11 +31,11 @@
...
   "scripts": {
-    "test:front": "cross-env IS_EE=true jest --config ./jest.config.front.js",
-    "test:front:watch": "cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
-    "test:front:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js",
-    "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
-    "lint": "eslint ."
+    "test:front": "cross-env IS_EE=true run -T jest --config ./jest.config.front.js",
+    "test:front:watch": "cross-env IS_EE=true run -T jest --config ./jest.config.front.js --watchAll",
+    "test:front:ce": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js",
+    "test:front:watch:ce": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js --watchAll",
```
Comment (innerdvations) [inline packages/plugins/color-picker/package.json:37]:
```suggestion
    "test:front": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js",
    "test:front:watch": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
    "test:front:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js",
    "test:front:watch:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
```

File: packages/core/admin/package.json [hunk 2]
```diff
@@ -30,14 +30,14 @@
...
-    "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
-    "test:front:ce:cov": "cross-env IS_EE=false jest --config ./jest.config.front.js --coverage",
-    "lint": "eslint ."
+    "test:unit": "run -T jest",
+    "test:unit:watch": "run -T jest --watch",
+    "test:front": "cross-env IS_EE=true run -T jest --config ./jest.config.front.js",
+    "test:front:watch": "cross-env IS_EE=true run -T jest --config ./jest.config.front.js --watchAll",
+    "test:front:ce": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js",
+    "test:front:watch:ce": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js --watchAll",
+    "test:front:ce:cov": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js --coverage",
```
Comment (innerdvations) [inline packages/core/admin/package.json:39]:
```suggestion
    "test:front": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js",
    "test:front:watch": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
    "test:front:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js",
    "test:front:watch:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
    "test:front:ce:cov": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js --coverage",
```

File: packages/core/email/package.json [hunk 3]
```diff
@@ -20,11 +20,11 @@
...
   "scripts": {
-    "test:front": "cross-env IS_EE=true jest --config ./jest.config.front.js",
-    "test:front:watch": "cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
-    "test:front:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js",
-    "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
-    "lint": "eslint ."
+    "test:front": "cross-env IS_EE=true run -T jest --config ./jest.config.front.js",
+    "test:front:watch": "cross-env IS_EE=true run -T jest --config ./jest.config.front.js --watchAll",
+    "test:front:ce": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js",
+    "test:front:watch:ce": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js --watchAll",
```
Comment (innerdvations) [inline packages/core/email/package.json:26]:
```suggestion
    "test:front": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js",
    "test:front:watch": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
    "test:front:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js",
    "test:front:watch:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
```

File: packages/core/upload/package.json [hunk 4]
```diff
@@ -16,13 +16,13 @@
...
-    "test:front:watch": "cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
-    "test:front:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js",
-    "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
-    "lint": "eslint ."
+    "test:unit": "run -T jest",
+    "test:unit:watch": "run -T jest --watch",
+    "test:front": "cross-env IS_EE=true run -T jest --config ./jest.config.front.js",
+    "test:front:watch": "cross-env IS_EE=true run -T jest --config ./jest.config.front.js --watchAll",
+    "test:front:ce": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js",
+    "test:front:watch:ce": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js --watchAll",
```
Comment (innerdvations) [inline packages/core/upload/package.json:24]:
```suggestion
    "test:front": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js",
    "test:front:watch": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
    "test:front:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js",
    "test:front:watch:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
```

File: packages/plugins/users-permissions/package.json [hunk 5]
```diff
@@ -20,13 +20,13 @@
...
-    "test:front:watch": "cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
-    "test:front:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js",
-    "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
-    "lint": "eslint ."
+    "test:unit": "run -T jest",
+    "test:unit:watch": "run -T jest --watch",
+    "test:front": "cross-env IS_EE=true run -T jest --config ./jest.config.front.js",
+    "test:front:watch": "cross-env IS_EE=true run -T jest --config ./jest.config.front.js --watchAll",
+    "test:front:ce": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js",
+    "test:front:watch:ce": "cross-env IS_EE=false run -T jest --config ./jest.config.front.js --watchAll",
```
Comment (innerdvations) [inline packages/plugins/users-permissions/package.json:28]:
```suggestion
    "test:front": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js",
    "test:front:watch": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
    "test:front:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js",
    "test:front:watch:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
```

**Review by innerdvations — COMMENTED** [review]
`cross-env` is only at the root level, not in any subpackage dev dependences, so `run` needs to come first or else none of these commands work when you're in this path

File: packages/plugins/users-permissions/package.json (hunk 5 above)
Comment (alexandrebodin) [reply packages/plugins/users-permissions/package.json:28]: nice cactch

**Review by innerdvations — APPROVED** [review]
LGTM

**Pull request merged**